src/Core/Resources/SharedResources.en.resx

```diff
@@ -546,7 +546,7 @@
     <value>Restrict users from being able to join any other organizations.</value>
   </data>
   <data name="SingleOrganizationPolicyWarning" xml:space="preserve">
-    <value>Organization members who are already a part of another organization will be removed from this organization and will receive an email notifying them about the change.</value>
+    <value>Organization Users and Managers who are already a part of another organization will be removed from this organization and will receive an email notifying them about the change.</value>
```

Comment:
Same... `"Organization members who are not Owners or Administrators..."`

This ensures as more roles are added in the future we're not limiting our verbiage and the language is around exclusivity vs. inclusivity, the excluded list will change far less frequently than the included list.

Reply:
Done.